akanon/Rilgor_Plegnog.pl

```diff
@@ -21,6 +25,23 @@ sub EVENT_SAY {
 		#:: Give them 51122 - Tradeskill Basics : Volume II
 		quest::summonitem(51122);
 	}
+	elsif ($text=~/hail/i) {
+		quest::say("Oh, what is it now. Can't you see that I am a very busy gnome?");
+	}
+	#:: Verify player has item 52947 Illegible Tome when asking about a 'translating device'
+	elsif ($text=~/translating device/i and plugin::check_hasitem($client, 52947)) {
+		quest::say("Rilgor Plegnog rubs his chin thoughtfully. 'A translating device? Now that is something rather complicated. It would take all of my skill as both a tinkerer and an enchanter. This is not the sort of thing I would do for nothing, no no no. I would need some sort of [compensation] for a project like that!'");
+	}
+	#:: Verify player has item 52947 Illegible Tome when asking about 'compensation'
+	elsif ($text=~/compensation/i and plugin::check_hasitem($client, 52947)) {
+		quest::say("'Hmm, well now that you mention it, some fine silks might do rather nicely. Oh yes, indeed. I do love my robe ever so much, but well, the rest of me could use some sprucing up. I always did feel a bit left out that I never picked up some of that nice Insidious");
+	}
+	#:: Verify player has item 52947 Illegible Tome when asking about 'insidious'
+	elsif ($text=~/insidious/i and plugin::check_hasitem($client, 52947)) {
```

Comment:
Should you be able to get multiple items, maybe need to check if NPC has it before summoning a new

Reply:
If player has given to NPC previously? Like with a data bucket flag?